tests/monotouch-test/ObjCRuntime/RegistrarTest.cs

```diff
@@ -2202,7 +2204,7 @@ public void WillEndDragging (UIScrollView scrollView, PointF velocity, ref Point
 		}
 #endif // !__WATCHOS__
 
-#if !__TVOS__ && !__WATCHOS__ && !MONOMAC// No ABPeoplePickerNavigationControllerDelegate
+#if HAS_ADDRESSBOOKUI
```

Comment:
Same, dual condition?

```suggestion
#if HAS_ADDRESSBOOK && HAS_ADDRESSBOOKUI
```